ground/openpilotgcs/src/plugins/config/cfg_vehicletypes/configmultirotorwidget.cpp

```diff
@@ -787,7 +787,7 @@ QString ConfigMultiRotorWidget::updateConfigObjectsFromWidgets()
 
             // Tricopter : Yaw mix slider value applies to servo (was fixed)
             // Get absolute MixerValueYaw, no servo reverse when Reverse All Motors is checked
-            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(getMixerValue(mixer, "MixerValueYaw")) * 1.27);
+            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(int(getMixerValue(mixer, "MixerValueYaw"))) * 1.27);
```

Comment:
Maybe I'm too cautious, but I'm afraid that changing this to fabs would cause some problems in the mixers. We spent a lot of time trying to fix previous off-by-one errors in the mixer GUI.